src/locales/en.i18n.json

```diff
@@ -0,0 +1,40 @@
+{
+  "common": {
+    "logIn": "Log in",
+    "signUp": "Sign up"
+  },
+  "auth": {
+    "login": {
+      "forgotPassword": "I forgot my password",
+      "dontHaveAccount": "Don't have an account?",
+      "userId": {
+        "label": "Email or telephone number",
+        "placeholder": "Type your email or telephone"
+      },
+      "password": {
+        "label": "Password",
+        "placeholder": "Type your password"
+      }
+    },
+    "register": {
+      "userId": {
+        "label": "Email or telephone number",
+        "placeholder": "Type your email or telephone"
+      },
+      "fullName": {
+        "label": "Full Name",
+        "placeholder": "Type your full name"
+      },
+      "userName": {
+        "label": "User Name",
+        "placeholder": "Type your user name"
+      },
+      "password": {
+        "label": "Password",
+        "placeholder": "Type your password"
+      },
+      "termsAndConditionsText": "By signing up, you accept our <1>Conditions</1>, the <3>Data Policy</3> and the <5>Cookies Policy</5>.",
```

Comment:
And here:
```suggestion
      "termsAndConditionsText": "By signing up, you accept our <0>Conditions</0>, the <1>Data Policy</1> and the <2>Cookies Policy</2>.",
```

Reply:
Aah, that’s way simpler. Thanks!

Reply:
![drake-laptop-drake](https://user-images.githubusercontent.com/67912621/166817328-4e01c172-63e0-4f0a-81cf-a38cdbb78eb2.gif)